types/src/front/files.ts

```diff
@@ -143,7 +147,7 @@ export function isSupportedImageContentType(
 
 export type FileStatus = "created" | "failed" | "ready";
 
-export type FileUseCase = "conversation" | "avatar" | "tool_output";
+export type FileUseCase = "conversation" | "avatar" | "tool_output" | "folder";
```

Comment:
Not sure `folder` is the best name for the useCase.

Reply:
Agreed, but keeping the naming until we refactor the use cases management to e.g. decouple 'allowed files' and 'processing strategy'., to have a clearer view. Unless you are very strongly opinionated.